src/views/Dashboard/Customer/Home/RequestTest/PaymentReceiptDialog.vue

```diff
@@ -178,6 +194,8 @@ export default {
   },
 
   async mounted () {
+    const txWeight = await getWeightTx(this.api, this.wallet, this.selectedService.labId)
+    this.txWeight = (txWeight.weight / 10000000)
```

Comment:
better to put unit measurement for each weight (e.g mili, nano, mio, etc) rather than hardcoded. is this hardcoded or do i get it wrong?

Reply:
remove this and use web3.utils to convert the amount